src/commands/math/commands.js

```diff
@@ -767,3 +767,25 @@ var Embed = P(Symbol, function(_, super_) {
     this.latex = options.latex || noop;
   }
 });
+
+//cmds is object: {symbol:[symboltype, latex, entity]}
+//where symboltype is "VanillaSymbol", "BinaryOperator", or "Variable"
+optionProcessors.addCommands = function(cmds) {	
+  for (var str in cmds) {
+    if (LatexCmds.hasOwnProperty(str)) {
+      throw '"' + str + '" is a built-in operator name';
```

Comment:
Maybe this should be `has already been defined` since it may have been set by a previous `.addCommands()` call?